packages/vda-sbt-client/test/blockchain-api.test.ts

```diff
@@ -0,0 +1,198 @@
+require('dotenv').config();
+import { dids, getBlockchainAPIConfiguration } from "./utils"
+import BlockchainApi from "../src/blockchain/blockchainApi"
+import { ethers, Wallet } from "ethers";
+import EncryptionUtils from "@verida/encryption-utils";
+
+const assert = require('assert')
+
+const did = dids[0];
+  
+
+const configuration = getBlockchainAPIConfiguration();
+const createBlockchainAPI = (did: any) => {
+    return new BlockchainApi({
+        identifier: did.address,
+        signKey: did.privateKey,
+        chainNameOrId: "testnet",
+        ...configuration
+    })
+}
+
+describe('vda-sbt-client blockchain api', () => {
```

Comment:
I'm getting timeouts for all of these. Can you set timeout to be 20 seconds for each test?

I tried setting `this.timeout()`, however it won't work ... which is weird as it works on all our other test suites.

Reply:
Is working locally now.